spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -427,20 +427,23 @@ def delete_provisioning_profile!(profile_id, mac: false)
       parse_response(r)
     end
 
-    def repair_provisioning_profile!(profile_id, name, distribution_method, app_id, certificate_ids, device_ids, mac: false)
+    def repair_provisioning_profile!(profile_id, name, distribution_method, app_id, certificate_ids, device_ids, mac: false, sub_platform: nil)
```

Comment:
It should be nil (or not existent) in all existing functionality, as you dont need a sub platform for iOS only tvOS.